src/helpers/loadOptimize.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { Store } from '../app/types';
+import { assertWindow } from '../app/utils';
+import createTestStore from '../test/createTestStore';
+import loadOptimize from './loadOptimize';
+
+describe('loadOptimize', () => {
+  let store: Store;
+  let window: Window;
+
+  beforeEach(() => {
+    store = createTestStore();
+    window = assertWindow();
+    window.gtag = jest.fn();
+    window.dataLayer = [];
+
+    if (typeof document === 'undefined') {
+      throw new Error('JSDom not loaded');
+    }
```

Comment:
You could use `assertDocument`, similar to `assertWindow` - or just use `window.document`